.github/pull_request_template.md

```diff
@@ -0,0 +1,61 @@
+### Description
+
+<!--
+A description of the change in this PR.
+Place a reference to the Github issue or the PagerDuty incident, if any, here.
+-->
+
+### Type of Change
+
+- [X] Routine: low risk change
+- [ ] Declared: a significant or high risk change
+- [ ] Emergency: a change made to fix an on-going incident
+
+<!--
+Classify this change as one of the following, according to the change management
+policy in https://drive.google.com/drive/folders/1YgEceW5HQ2bQ4OAuhFIgN1oHmLk9RfTV
+-->
+
+### Testing
+
+- [ ] None: Implicit through tooling or non-configuration management change
+- [ ] Unit Tests
+- [ ] Functional Tests
+- [X] Manual Tests
+
+<!--
+Describe the testing procedure that has been done, or will be done, if any.
+Optional for routine changes, mandatory for declared changes.
+-->
+
+### Rollout Procedure
+
+- [ ] None: Non-configuration management change
+- [ ] Manual Roll-Out: <!-- If checked, replace with description -->
+- [X] Automated Roll-Out (Full): Changes will be applied when an image is built from master and pushed to redcanary/cb-event-forwarder:latest
+
+<!--
+Describe how this change will be rolled out. For most changes in this repository,
+the following default should suffice.
+-->
+
+### Rollback Procedure
+
+Reverting the merge of this PR, and applying the rollout procedure above, will roll
```

Comment:
This isnt the case for code, as when code is reverted, a new image would be built, and then THAT would be rolled out. I believe that we really just need to document the roll-out procedure for docker images within a kubernetes clusters (e.g., process, tooling, etc), and then reference that in here.

Reply:
Since we don't really have a roll-out for this repo, I'm not sure what to change this to.  Would leaving as is be alright for now rather than having no template?

Reply:
Just explaining that roll-back would be to revert the PR, and a new image will be built, should be enough.